packages/validation/src/validateRecord/RecordValidationError.ts

```diff
@@ -25,7 +25,7 @@ import type {
 
 export type FieldDetails = {
 	fieldName: string;
-	value: DataRecordValue;
+	fieldValue: DataRecordValue;
```

Comment:
Property name change for the `RecordValidationError` shared type `FieldDetails`.